JacobF94/NSS-Frontend-Capstone
# initial MVP version of app

#### Changes Made
1. Added photos to public directory
2. finished all routing between modules
3. Day module correctly displays notes alongside chosen exercises
4. create note button toggles the note form and itself to switch
5. Select dropdown box for exercise selections properly updates itself
6. finished homepage button links and current day display
7. exercise detail page includes written information as well as a photo
​
#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout jf-addPhotos
    ```
2. Open a new Terminal tab (⌘T or Ctrl+Shift+T) and navigate to the server directory.
3. Test app functionality.
    > Instructions for how reviewer can test functionality, and detailed description of what the expected outcome is.
    > Example: When user does BLANK, then BLANK should happen.
4. View code file.
    > Confirm file modifications are present as indicated above.
    > Confirm no unused code or extraneous comments exist.
